web/src/beta/components/PublishStateSwitchField/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+import React, { ReactNode } from "react";
+
+import PublishStateSwitchField from ".";
+
+export default {
+  component: PublishStateSwitchField,
+} as Meta;
+
+type Story = StoryObj<typeof PublishStateSwitchField>;
+
+const Wrapper: React.FC<{ children: ReactNode }> = ({ children }) => (
+  <div style={{ width: "174px", height: "24px" }}>{children}</div>
+);
+export const Default: Story = {
+  args: {
+    list: ["Unpublished", "Published"],
+    onChange: action("onchange"),
```

Comment:
you don't need manually pass this function.
If the field name matches [this Regex](https://github.com/reearth/reearth/blob/main/web/.storybook/preview.tsx#L39), it will be automatically attached.